changelogs/fragments/3327-tss-token-authorization.yml

```diff
@@ -0,0 +1,5 @@
+minor_changes:
+  - tss lookup plugin - added ``token`` parameter for token authorization;
+    ``username`` and ``password`` are optional when ``token`` is provided;
+    updated examples to include ``token`` and ``domain`` parameters
```

Comment:
```suggestion
    ``username`` and ``password`` are optional when ``token`` is provided
```
Documentation changes should not be mentioned in the changelog (only exception: major bugfixes in the documentation).